src/lib.rs

```diff
@@ -619,8 +618,62 @@ macro_rules! bitflags {
                 }
             }
 
+            __impl_public_bitflags_forward! {
+                $BitFlags: $T, InternalBitFlags
+            }
+
+            __impl_public_bitflags_iter! {
+                $BitFlags
+            }
+        };
+
+        bitflags! {
+            $($t)*
+        }
+    };
+    (
+        impl $BitFlags:ident: $T:ty {
+            $(
+                $(#[$inner:ident $($args:tt)*])*
+                const $Flag:ident = $value:expr;
+            )*
+        }
+
+        $($t:tt)*
+    ) => {
+        __impl_public_bitflags_consts! {
+            $BitFlags: $T {
+                $(
+                    $(#[$inner $($args)*])*
+                    $Flag = $value;
+                )*
+            }
+        }
+
+        #[allow(
+            dead_code,
+            deprecated,
+            unused_doc_comments,
+            unused_attributes,
+            unused_mut,
+            unused_imports,
+            non_upper_case_globals
+        )]
+        const _: () = {
+            // NOTE: We don't need to check `__is_primitive` here because we don't derive
```

Comment:
Out-of-date: This is now a trait implemented by declaring the internal bits field.